sass/includes/_buttons.scss

```diff
@@ -5,9 +5,26 @@
     border: 0;
     padding: 1rem;
     margin: 0.5rem;
+}
 
-    &--no-underline {
-        @extend .tna-button;
-        text-decoration: none;
-    }
-}
\ No newline at end of file
+.tna-button--dark, .tna-button--dark:visited {
+  display: inline-block;
+  text-decoration: none;
+  background-color: $color__dark;
+  color: $color__white;
+  outline: 0.1875rem solid $color__dark;
+  padding: 1rem;
+  margin: 0.5rem;
+
+  &:hover {
+    background-color: $color__white;
+    color: $color__dark;
+    outline: 0.1875rem solid $color__dark;
+  }
+
+  &:focus {
+    outline: 0.312rem solid $color__blue_4;
+    outline-offset: 0.125rem;
+    z-index: 999;
```

Comment:
Quick question: what's the reason for `z-index` on the `:focus` state here?

Reply:
My mistake, I'll remove it. I think I copied it from an existing button style and forgot to remove it when updating the styles.